examples/launch_rl_tuning.py

```diff
@@ -0,0 +1,49 @@
+"""
```

Comment:
Uups, is this really new here?

Reply:
Indeed, the diff should not appear. Can you make sure that you have not duplicated files with `git ls-files`?